Distribution/Server/Features/Security/FileInfo.hs

```diff
@@ -35,12 +36,13 @@ import qualified Hackage.Security.Server as Sec
 
 -- | Simplified form of the FileInfo used in hackage-security
 data FileInfo = FileInfo {
-    fileInfoLength :: Sec.Int54
+    fileInfoLength :: !Sec.Int54
   , fileInfoSHA256 :: SHA256Digest
+  , fileInfoMD5    :: MD5Digest
   }
   deriving (Typeable, Show, Eq)
 
-deriveSafeCopy 0 'base ''FileInfo
+deriveSafeCopy 1 'base ''FileInfo
```

Comment:
I incremented the SafeCopy version, but I have no idea what else needs to be done here